pkgs/applications/misc/keepass-plugins/keeotp2/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, buildEnv, fetchurl, mono }:
+
+let
+  version = "1.5.9";
+  drv = stdenv.mkDerivation {
+    pname = "keeotp2";
+    inherit version;
+
+    src = fetchurl {
+      url    = "https://github.com/tiuub/KeeOtp2/releases/download/v${version}/KeeOtp2.plgx";
+      sha256 = "f5c797cea710e0cd170c47133d6144faa8d14b9c3d1e045da2d08c82682990c9";
+    };
+
+    meta = with lib; {
+      description = "KeeOtp2 is a plugin for KeePass. It provides a form to display one time passwords and is fully compatible with the built-in OTP function.";
+      homepage    = "https://github.com/tiuub/KeeOtp2";
+      platforms   = with lib.platforms; linux;
```

```suggestion
      platforms   = lib.platforms.linux;
```

You would also think about why this is the platform and then use the constraints by the used packages. For example, `keepass.meta.platforms` (takes away part of your responsibility).

Especially, I see that the plugin repo does not limit to Linux, and afaik also keepass doesn't.